packages/pancake-uikit/src/components/Text/TooltipText.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import Text from "./Text";
+
+const TooltipText = styled(Text)`
+  text-decoration: ${({ theme }) => `underline dotted ${theme.colors.textSubtle}`};
+  text-underline-offset: 0.1em;
+`;
+
+export default TooltipText;
```

Comment:
This is exactly what I was thinking...but exported from the Tooltip. No worries, we can think about alternative solutions later.